src/components/content/about/detailContainer/detailContainerData.js

```diff
@@ -0,0 +1,31 @@
+import aboutBanner1 from "../../../../images/banners/about-page/about-banner-1.png";

```

Comment:
ti am aratat data trecuta cum sa muti pozele in public folder si sa pui calea direct in loc sa le dai import asa. 

tot atunci am vb despre cum toooot continutu de pe peste tot intr-un loc comun (de preferat json) si sa l trimiti prin props

Reply:
Mi-ai spus ca imaginile trebuie sa fie in public, dar nu mi-ai aratat cum, am incercat eu sa fac dar mi-a bubuit aplicatie de 2 ori si am lasat-o balta pana sa ne vedem iar